shell/packages/sandstorm-db/db.js

```diff
@@ -498,6 +498,9 @@ Notifications = new Mongo.Collection("notifications");
 //   text:         The JSON-ified LocalizedText to display in the notification.
 //   isUnread:     Boolean indicating if this notification is unread.
 //   timestamp:    Date when this notification was last updated
+//   initiatingIdentity: Identity ID of the user who initiated this notification.
```

Comment:
Hmm, it's not clear to me that there's any reason to limit `initiatingIdentity` to be present only together with `grainId`, even if that is the case currently.